docs/secure-connections/sauce-connect-5/guides/localhost-proxying.md

```diff
@@ -0,0 +1,62 @@
+---
+id: localhost-proxying
+title: Proxying Localhost with Sauce Connect 5
+sidebar_label: Proxying Localhost
+---
+
+Sauce Connect 5 enables your local applications and services to be securely accessed from the Sauce Labs cloud. However, special care is required when working with `localhost` addresses, particularly when testing from mobile devices or when using real devices in Sauce Labs.
+
+## Enabling Localhost Proxying
+
+To route localhost traffic through Sauce Connect, you must enable the [`--proxy-localhost`](/dev/cli/sauce-connect-5/run#proxy-localhost) option when starting the tunnel.
+
+This option controls how Sauce Connect handles requests to localhost (or 127.0.0.1):
+
+- `--proxy-localhost allow`: Proxies all localhost requests through Sauce Connect to your local machine. Use this mode when you want devices in Sauce Labs to access services running on your machine (for example, http://localhost:3000).
+- `--proxy-localhost direct`: Allows localhost requests but skips proxying them through any configured upstream proxy. Use this your upstream proxy localhost should not be exposed.
```

Comment:
I think you're missing a word - "Use this **when/if** [...]"

Reply:
💯